specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintAssignment.json

```diff
@@ -345,6 +345,13 @@
           "items": {
             "type": "string"
           }
+        },
+        "excludedActions": {
```

Comment:
>excludedActions [](start = 9, length = 15)

Should we add something in the description that this will automatically include Microsoft.Network/virtualNetworks/subnets/join/action in some cases?

Reply:
Agree, I'll add the default excludedActions added based on the lock mode.